packages/site-server/src/lib/catalog/routes/import-page.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// This must be imported before lit
```

Comment:
Remove comment (seems like it's in the wrong place, and it's not true anymore anyway because of the new built-in SSR polyfilling)